Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mhart/alpine-node:5.0
+
+ENV BUILD_PACKAGES bash curl-dev ruby-dev build-base libffi-dev libpq postgresql-dev git
+ENV RUBY_PACKAGES ruby ruby-io-console ruby-bundler ruby-raindrops ruby-nokogiri
+
+# Update and install all of the required packages.
+# At the end, remove the apk cache
+RUN apk update && \
+    apk upgrade && \
+    apk add $BUILD_PACKAGES && \
+    apk add $RUBY_PACKAGES && \
+    rm -rf /var/cache/apk/*
+
+RUN mkdir /app
+WORKDIR /app
+
+ADD . /app
+RUN bundle install --jobs 4 --without development test
```

Comment:
Ah, that's interesting. It does indeed load a lot of things...

It's not a big deal, it's probably just a matter of a minute extra in the build process, and this app is not critical.